exonum/src/helpers/fabric/info.rs

```diff
@@ -0,0 +1,96 @@
+// Copyright 2018 The Exonum Team
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! This module implements information request commands.
+
+use serde_json;
+
+use std::collections::HashMap;
+
+use super::{internal::{CollectedCommand, Command, Feedback},
+            Argument,
+            CommandName,
+            Context};
+
+// Context entry for the type of the requested information.
+const INFO_REQUEST: &str = "INFO_REQUEST";
+
+/// Information request command. Supported sub-commands:
+///
+/// - `core-version` - prints the version of the Exonum core.
+/// - `list-services` - prints the list of the services the node is build with.
+pub struct Info {
+    pub services: Vec<String>,
+}
+
+impl Info {
+    /// Creates a new `Info` instance.
+    pub fn new(services: Vec<String>) -> Self {
+        Self { services }
+    }
+
+    /// Returns the name of the `Maintenance` command.
+    pub fn name() -> CommandName {
```

Comment:
Is this method vital? It is only relevant for trait implementation, isn't it?

Reply:
I like the idea. :+1: